client/src/components/modal.jsx

```diff
@@ -0,0 +1,18 @@
+import styles from './styles.css.js';
+
+const Modal = props => {
+  if (!props.modalActive) {
+    return null;
+  } else {
+    return (
+      <div style={styles.modal}>
+        <div style={styles.modalContent}>
+          <span style={styles.modalBtn} onClick={()=>{props.closeModal()}}>&times;</span>
```

Comment:
I'm not sure if this is for some reason not a preferable way to do it, but i was able to get this to work by just saying `onClick={props.closeModal}`

Reply:
Thanks!!! I'll try this out and change it!